fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -212,13 +214,31 @@ from tango.server import Device, DeviceMeta
 «IF !cls.classProperties.empty || !cls.deviceProperties.empty»from tango.server import «IF !cls.classProperties.empty»class_property, «ENDIF»«IF !cls.deviceProperties.empty»device_property«ENDIF»«ENDIF»
 from tango import AttrQuality, DispLevel, DevState
 from tango import AttrWriteType, PipeWriteType
+from enum import IntEnum
 «cls.inheritedAdditionalImportHL»
 # Additional import
 «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«cls.protectedAreaHL("additionnal_import")»«ENDIF»
 
 __all__ = ["«cls.name»", "main"]
 '''
 
+def enumClasses(PogoDeviceClass cls) '''
+«IF cls.attributes!==null»
+«FOR attr:cls.attributes»
+«IF attr.dataType.pythonTypeHL.equalsIgnoreCase("'DevEnum'")»
+
+
+class «attr.name.toFirstUpper»(IntEnum):
+    """«attr.name.toFirstUpper» enum."""

```

Comment:
The issue #80 suggested a slightly more verbose docstring.  That would be better.
`"""Python enumerated type for «attr.name» attribute."""`

Reply:
Okay. Updated the docstring.